src/tsconfig.json

```diff
@@ -20,6 +20,6 @@
     "**/test/**/*.js",
     "test-util/**/*.js",
     "karma.config.js",
-    "../frontend-shared/src/test/**/*.js",
+    "../frontend-shared/src/**/*-test.js",
```

Comment:
We have occassionally put non-test utility modules inside test directories. Hence the other exclusion patterns exclude directories named `test` rather than files with a `-test.js` suffix. I'd suggest to do the same here, but just allow the `test` directory to be anywhere under `../frontend-shared/`.